src/SFA.DAS.Commitments.Support.SubSite/Orchestrators/ApprenticeshipsOrchestrator.cs

```diff
@@ -0,0 +1,236 @@
+﻿using FluentValidation;
+using MediatR;
+using SFA.DAS.Commitments.Application.Exceptions;
+using SFA.DAS.Commitments.Application.Queries.GetApprenticeship;
+using SFA.DAS.Commitments.Application.Queries.GetApprenticeshipsByUln;
+using SFA.DAS.Commitments.Application.Queries.GetCommitment;
+using SFA.DAS.Commitments.Domain;
+using SFA.DAS.Commitments.Support.SubSite.Mappers;
+using SFA.DAS.Commitments.Support.SubSite.Models;
+using SFA.DAS.HashingService;
+using SFA.DAS.NLog.Logger;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Orchestrators
+{
+    public class ApprenticeshipsOrchestrator : IApprenticeshipsOrchestrator
+    {
+        private readonly ILog _logger;
+        private readonly IMediator _mediator;
+        private readonly IValidator<ApprenticeshipSearchQuery> _searchValidator;
+        private readonly IHashingService _hashingService;
+        private readonly IApprenticeshipMapper _apprenticeshipMapper;
+        private readonly ICommitmentMapper _commitmentMapper;
+
+        public ApprenticeshipsOrchestrator(ILog logger,
+                                            IMediator mediator,
+                                            IApprenticeshipMapper apprenticeshipMapper,
+                                            IValidator<ApprenticeshipSearchQuery> searchValidator,
+                                            IHashingService hashingService,
+                                            ICommitmentMapper commitmentMapper)
+        {
+            _logger = logger ?? throw new ArgumentException(nameof(logger));
```

Comment:
Null checks aren't necessary in my opinion.

Reply:
Removed null checks